docs/platform-integration/essentials/file-picker.md

```diff
@@ -60,72 +73,51 @@ async Task<FileResult> PickAndShow(PickOptions options)
         var result = await FilePicker.PickAsync(options);
         if (result != null)
         {
-            Text = $"File Name: {result.FileName}";
             if (result.FileName.EndsWith("jpg", StringComparison.OrdinalIgnoreCase) ||
                 result.FileName.EndsWith("png", StringComparison.OrdinalIgnoreCase))
             {
-                var stream = await result.OpenReadAsync();
-                Image = ImageSource.FromStream(() => stream);
+                using var stream = await result.OpenReadAsync();
+                var image = ImageSource.FromStream(() => stream);
             }
         }
-        
+
         return result;
     }
     catch (Exception ex)
     {
         // The user canceled or something went wrong
     }
-    
+
     return null;
 }
 ```
 
-Default file types are provided with `FilePickerFileType.Images`, `FilePickerFileType.Png`, and `FilePickerFilerType.Videos`. You can specify custom files types when creating the `PickOptions` and they can be customized per platform. For example here is how you would specify specific comic file types:
+Default file types are provided with `FilePickerFileType.Images`, `FilePickerFileType.Png`, and `FilePickerFilerType.Videos`. You can specify custom file types per platform, by creating an instance of the `FilePickerFileType` class. The constructor of this class takes a dictionary that is keyed by the `DevicePlatform` type to identify the platform. The value of the dictionary key is a collection of strings representing the file types. For example here is how you would specify specific comic file types:
 
 ```csharp
-var customFileType =
-    new FilePickerFileType(new Dictionary<DevicePlatform, IEnumerable<string>>
-    {
-        { DevicePlatform.iOS, new[] { "public.my.comic.extension" } }, // or general UTType values
-        { DevicePlatform.Android, new[] { "application/comics" } },
-        { DevicePlatform.UWP, new[] { ".cbr", ".cbz" } },
-        { DevicePlatform.Tizen, new[] { "*/*" } },
-        { DevicePlatform.macOS, new[] { "cbr", "cbz" } }, // or general UTType values
-    });
-var options = new PickOptions
+var customFileType = new FilePickerFileType(new Dictionary<DevicePlatform, IEnumerable<string>>
+                    {
```

Comment:
Is the code deliberately aligned like this? At best it looks one char out on the review site.